linux_os/guide/services/mail/service_postfix_enabled/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhel9,sle15
+prodtype: ol7,ol8,rhel7,rhel8,rhel9,sle12,sle15
```

Comment:
Not seeing any references is that by design?

Reply:
To me, it seemed like a dependency.

`2.2.19 Ensure mail transfer agent is configured for local-only mode (Automated)` uses checks related to it. The build failed locally if I didn't add the prodtype.